Refactor docs on `options.sanitize` in `readme.md` (#35)
remarkjs/remark-html: readme.md

###### `options.sanitize`

How to sanitize the output (`Object` or `boolean`, default: `true`):

> - If `false`, no HTML is sanitized, and dangerous HTML is left unescaped.
> - If `true` or an `object`, sanitation is done by [`hast-util-sanitize`][sanitize].
> - If an object is passed in, it’s given as a schema to `hast-util-sanitize`.
> - If `true`, input is sanitized according to [GitHub’s sanitation rules][github].

Reviewer: Great idea!

```suggestion
*   `false`
    — HTML is not sanitized, dangerous HTML persists
*   `true`
    — HTML is [`hast-util-sanitize`][sanitize] according to [GitHub’s sanitation
    rules][github], dangerous HTML is dropped
*   `Object`
    — the object is treated as a `schema` for how to sanitize with
    [`hast-util-sanitize`][sanitize], dangerous HTML is dropped
```

How about this? a) Reworded in a way that makes more sense to me, can you confirm, b) conforms to code style, c) dropped the fourth, duplicate, bullet, which was a bit confusing to me?

Author: yes this looks better for me :)
